lua/lspconfig/perlpls.lua

```diff
@@ -0,0 +1,32 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.perlpls = {
+  default_config = {
+    cmd = {"pls"};
+    settings = {
+      perl = {
+        perlcritic = { enabled = false };
+      };
+    };
+    filetypes = {"perl"};
+    root_dir = function(fname)
+      return util.root_pattern(".git")(fname) or vim.fn.getcwd()
```

Comment:
use util.path.dirname instead of cwd, see pyright

Reply:
done.